Add helper functions for testing

EshaanAgg opened the pull request:
**What kind of change does this PR introduce?**
Feature

**Issue Number:**
Fixes part of #961

**Did you add tests for your changes?**
Yes

**Summary**
Migrated the Organization tests to use helper functions.

**Does this PR introduce a breaking change?**
No

**Other information**
The maintainers can review the helpers added and all the tests in the `Organization` category and recommend other functions with may help to reduce duplication. Then we can start migrating these helpers to other tests.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes

anshgoyalevil commented:
@EshaanAgg Please use ```nanoid().toLowerCase()``` in place of ```password``` ```firstName``` ```lastName``` ```description``` ```name``` and all other string variables which expect a random string.

EshaanAgg commented on tests/helpers/userAndOrg.ts:
```diff
@@ -0,0 +1,64 @@
+  const testUser = await User.create({
+    email: `email${nanoid().toLowerCase()}@gmail.com`,
+    password: "password",
+    firstName: "firstName",
```
Changed in latest commit

EshaanAgg commented:
Thanks @anshgoyalevil for the cautionary advice. Haven't mocked Mongo ID's anywhere, so didn't have to implement that.
@palisadoes Migrated the static fields to nanoID and added prefix identifiers to all of them so that they are more verbose and easier to debug in case of errors.

anshgoyalevil commented:
> Thanks @anshgoyalevil for the cautionary advice. Haven't mocked Mongo ID's anywhere, so didn't have to implement that. @palisadoes Migrated the static fields to nanoID and added prefix identifiers to all of them so that they are more verbose and easier to debug in case of errors.

Your changes looks good, have you tried running the tests in local machine, and checked the coverage?

EshaanAgg commented:
I ran all the tests that I had changed locally and all of them passed. I am yet to check the coverage. 
Can you guide me as to how to check the coverage for just the files that I have changed?

anshgoyalevil commented:
> I ran all the tests that I had changed locally and all of them passed. I am yet to check the coverage. Can you guide me as to how to check the coverage for just the files that I have changed?

use the command ```npm run test fileNameWithExtension --coverage``` 
This will run the specific test which you want.
After that, go to the coverage folder inside the root directory, and check for the ```.html``` file for the specific test that you just ran. Make sure to open the ```fileNameWithExtension.html``` and not the ```fileNameWithExtension.spec.html``` file.

EshaanAgg commented:
Thanks for the help. 100% code coverage for all the migrated files!

anshgoyalevil commented:
> Thanks for the help. 100% code coverage for all the migrated files! ![image](https://user-images.githubusercontent.com/96648934/216022628-fc8f364f-5cda-4293-b9a2-db6327ee7a62.png)

Great
